content/TrueCommand/Administration/LDAPServers.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "LDAP Servers"
+weight: 40
+---
+
+{{< toc >}}
+
+Users can configure TrueCommand to use LDAP servers for security and authentication management among connected TruNAS systems.
+
+## Add an LDAP Server in TrueCommand
+
+Click the gear icon in the upper toolbar and select **Administration**. Scroll down to the **LDAP Servers** widget and click **ADD**.
+
+![AddLDAPServer](/images/TrueCommand/AddLDAPServer.png "Add LDAP Servers")
+
+| Field | Description |
+|---------|-------------|
+| **Hostname** | Hostname of LDAP/AD server, with optional port. e.g. example.com:636. |
+| **Domain** | LDAP base domain. e.g. dc=example,dc=com. |
+| **Group Domain** | Optional for admins that want to avoid issues with unwanted groups by forcing group searches to start at a deeper domain. |
+| **User ID Field** | Class mapped to login username. The default is uid. |
+| **Group ID Field** | Class for finding groups associated with a user. The default is cn. |
+| **BIND User Domain** | Full user domain for binding before finding user FQID. Optional. |
+| **Realm** | The realm that performs authentication against the LDAP server. |
+| **BIND Password** | If bind user is set, use this password when performing a simple bind on user search. |
+| **KDC** | The Kerberos Key Distribution Center that supplies session tickets and temporary session keys to users and computers within the LDAP server. |
+
+Once you fill the form according to your server, click **ADD SERVER**.
```

Comment:
Try to avoid using once unless saying "you only perform this once." Use after instead.
```suggestion
After you fill the form according to your server, click **ADD SERVER**.
```